_posts/2020-01-23-operating-system-and-browser-accessibility-display-modes.md

```diff
@@ -0,0 +1,373 @@
+    animation: none;
+  }
+}
+```
+
+## High Contrast Mode
+
+This mode prioritizes legibility of content above all else. When activated, it maximizes the contrast between OS content and the background it is placed on top of, even at the expense of the appearance of the UI. High Contrast Mode comes with preset color themes, and also allows you to create your own. This allows someone to select combinations of colors that work best for them.
+
+It is helpful for:
+
+- People who have low vision or photosensitive conditions,
+- and/or who are using their device in an environment with very bright light and glare.
+
+An example of one of these conditions could be someone with cataracts who wants to stay up to date on social media.
+
+<figure role="figure" aria-label="High Contrast Mode #1 theme.">
+  <img alt="Screenshot of High Contrast Mode running on Windows 10. A Wikipedia page for Bodiam Castle is in the background, displayed in Internet Explorer 11. In the foreground is a window showing Window’s File Explorer. Content for both the Operating System UI and web content have been updated to display high contrast color values." src="/img/posts/2020-01-23-operating-system-and-browser-accessibility-display-modes/high-contrast-mode.png">
+  <figcaption>High Contrast Mode #1 theme.</figcaption>
+</figure>
+
+### What it affects
+
+High Contrast Mode is **only** available for Windows (versions 7 and up). For web content, it will **only** work with Internet Explorer (versions 10 and 11), as well as Edge (any version). It affects **all** UI elements of Windows, as well as apps developed by Microsoft. It will also work for most third party apps. Most websites will have at least **partial** support, with support depending on how they are coded.
+
+### How to enable it
+
+Assumes the latest version of each Operating System or browser.
+
+#### Windows
+
+1. Go to **Settings**.
+1. Go to **Ease of Access**.
+1. Go to **High Contrast**.
+1. Select a theme from the **Choose a theme** dropdown and then activate the **Apply** button.
+
+### How to target it in code
+
+High Contrast Mode will automatically be applied to web content described semantically in HTML, so long as you are using Internet Explorer versions 10 or 11, or any version of Microsoft Edge. For example, paragraphs of text described using the `p` element will automatically update when High Contrast Mode is activated. It will also update the same way if you use the `a` element for links, the `button` element for actions, the `input` element for form controls, etc.
+
+In situations where the HTML cannot be modified, High Contrast Mode styling can be tweaked using the [`-ms-high-contrast` media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast) in CSS. In these situations, you should use a specialized subset of [CSS2 system color keywords](https://www.w3.org/TR/2018/REC-css-color-3-20180619/#css2-system), as High Contrast Mode theme colors are dynamic.
```

Comment:
i don't believe this works with latest edge now though, eh?

https://twitter.com/gregwhitworth/status/1217537932770177024?s=20

Reply:
I'd like to keep it, just for the sake of some backwards compatibility with aliasing. I'll update it when forced colors becomes a thing, but for now it's just an idea they're working on getting implemented in the browser.

Reply:
i think it's good to keep, i just think it's also worth mentioning the prefixed version only works in IE11 now.

Reply:
Cool. Just updated this and the other conversation. Cool to merge?